submissions/pozzitive11/memoty-pair-game/js/index.js

```diff
@@ -49,34 +49,38 @@ const gameContainer = document.querySelector(".game__container");
 gameContainer.innerHTML = gameCard.join("");
 const gameCards = document.querySelectorAll(".game__card");
 
-let counter = 0;
-let arr = [];
-
-function flipCard() {
-  if (this && this.classList.contains("game__card")) {
-    this.classList.add("flip");
-
-    arr.push(this);
-  }
-  checkArrLength();
-
-  winCOndition();
-}
+let numberOfMatches = 0;
+let ArrayOfFlippedCards = [];
+
+gameCards.forEach((card) =>
+  card.addEventListener("click", ({ target }) => {
+    const parentTarget = target.parentElement;
```

Comment:
Don't access elements via `parentTarget` or `siblings`, or etc. Someone can additional wrapper and your code will break. You can use `closest` instead.

Second. Instead of finding parent of clicked element you can catch event on this parent. And then you don't need to find a parent.